src/common/base-store.ts

```diff
@@ -37,12 +37,16 @@ export class BaseStore<T = any> extends Singleton {
     return path.basename(this.storeConfig.path);
   }
 
-  get path() {
-    return this.storeConfig.path;
+  get syncRendererChannel() {
+    return `store-sync-renderer:${this.name}`
+  }
+
+  get syncMainChannel() {
+    return `store-sync-main:${this.name}`
```

Comment:
This event-name should be more unique to not overlap from different extensions which use `ExtensionStore < BaseStore`.

```suggestion
    return `store-sync-main:${this.path}`
```

Reply:
Good catch!